### PR TITLE
CAPA: Release v25.1.1.

### PR DESCRIPTION
**Pull request by Gacko**
### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

**Comment by AndiDog** [comment]
Prerequisite for review before we _use_ the release in migrations: https://github.com/giantswarm/capa-iam-operator/pull/334

File: capa/v25.1.1/announcement.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v25.1.1 for CAPA is available**. This release updates cluster-aws to v1.1.1 which includes several important patches.
```
Comment (AndiDog) [inline capa/v25.1.1/announcement.md:1]:
```suggestion
**Workload cluster release v25.1.1 for CAPA is available**. This release updates cluster-aws to v1.1.1 which allows configuring the node CIDR mask size, toggle the auditd service, and define multiple service account issuers in a certain order (important for migration of Vintage clusters).
```

Comment (Gacko) [reply capa/v25.1.1/announcement.md:1]: I will never understand why English grammar adds a comma in enumerations. Apart from that: Thank you! ❤️

File: capa/v25.1.1/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v25.1.1 for CAPA :zap:
+
```
Comment (T-Kukawka) [inline capa/v25.1.1/README.md:2]: This release introduces several changes that are required for Vintage to CAPA migration use-cases. 

Most notable change is that now `auditd` kernel setting is *disabled* by default. If you actively use this feature, please add following setting `global.components.auditd.enabled` to `True` in the [cluster values file](https://github.com/giantswarm/cluster/blob/main/helm/cluster/values.yaml#L31) before the upgrade.

Comment (T-Kukawka) [inline capa/v25.1.1/README.md:2]: `This release introduces several changes that are required for Vintage to CAPA migration use-cases.

Most notable change is that now auditd kernel setting is disabled by default. If you actively use this feature, please add following field global.components.auditd.enabled set to true in the [cluster values file](https://github.com/giantswarm/cluster/blob/main/helm/cluster/values.yaml#L31) before the upgrade.`

File: capa/v25.1.1/README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # :zap: Giant Swarm Release v25.1.1 for CAPA :zap:
 
+This release introduces several changes that are required for Vintage to CAPA migration use-cases.
+
+Most notable change is that now `auditd` is *disabled* by default. If you actively use this feature, please add the following field `global.components.auditd.enabled` set to `true` in the [cluster values file](https://github.com/giantswarm/cluster/blob/main/helm/cluster/values.yaml#L31) before the upgrade.
```
Comment (AndiDog) [inline capa/v25.1.1/README.md:5]:
```suggestion
Most notable change is that now `auditd` is *disabled* by default. If you actively use this feature, please add the following field `global.components.auditd.enabled` set to `true` in the [cluster values file](https://github.com/giantswarm/cluster-aws/blob/main/helm/cluster-aws/README.md#components) before the upgrade.
```

**Review by AndiDog — APPROVED** [review]

**Comment by Gacko** [comment]
Closing in favor of https://github.com/giantswarm/releases/pull/1417.

**Pull request closed without merging**